src/inmanta_ext/core/extension.py

```diff
@@ -50,3 +51,4 @@ def setup(application: ApplicationContext) -> None:
     application.register_slice(orchestrationservice.OrchestrationService())
     application.register_slice(dryrunservice.DyrunService())
     application.register_slice(notificationservice.NotificationService())
+    application.register_slice(environment_metrics_service.EnvironmentMetricsService())
```

Comment:
@arnaudsjs Am I right that is was missing? I needed this in LSM

Reply:
Yes. Indeed. This is required. Otherwise the server won't start the slice.